packages/ui/src/components/NavBar/NavBar.tsx

```diff
@@ -11,17 +13,22 @@ const NavBar = React.memo(function NavBar(props: {
 }) {
   const { bottom } = useSafeAreaInsets();
 
+  const theme = useTheme();
+
   return (
     <View
       position="absolute"
       width={'100%'}
       bottom={0}
-      backgroundColor={'$background'}
-      borderTopWidth={1}
       paddingTop={'$m'}
       borderTopColor={'$border'}
       height={navHeight + bottom}
     >
+      <BlurView
```

Comment:
This will likely cause issues on Android. Other places where we're using `BlurView` have to be special cased for iOS only.

Reply:
Thank you for resolving!